### ISSUE TITLE
New issue

### ISSUE THREAD

**Issue by FallenAzrael1** [Anistream/Anistream-Mobile#129]
At first, the app won't open and kept crashing. I came to re-download and saw all these threads saying the same thing.
I re-downloaded the app and tried to run it.
It didn't crash but it brought up a popup saying "Something is blocking your connection, please use a VPN".
I turn on my VPN and run the app, still got the same error.

**Comment by itachi0713** [comment]
> At first, the app won't open and kept crashing. I came to re-download and saw all these threads saying the same thing.
> I re-downloaded the app and tried to run it.
> It didn't crash but it brought up a popup saying "Something is blocking your connection, please use a VPN".
> I turn on my VPN and run the app, still got the same error.

Yeah like one of the problem is that you cannot change the quality of the anime... it's fixed with 720p and 1080p

**Comment by PikaPunch1** [comment]
I just keep loading and it doesn't even give me the VPN Message
